# 01_usr_bin.diff by Ryan Kavanagh <ryanakca@kubuntu.org>
# Description: Use /usr/bin, not /usr/local/bin
Index: basic256/BASIC256.pro
===================================================================
--- basic256.orig/BASIC256.pro	2010-11-16 12:07:02.870181279 -0500
+++ basic256/BASIC256.pro	2010-11-16 12:08:24.715175192 -0500
@@ -86,7 +86,7 @@
 	transFiles.files = ./Translations/*.qm
 	transFiles.path = /usr/share/basic256
 	INSTALLS += transFiles
-	target.path = /usr/local/bin
+	target.path = /usr/bin
 	INSTALLS += target
 
 }
